Package.swift

```diff
@@ -17,11 +17,11 @@ let package = Package(
   name: "swift-testing",
 
   platforms: [
-    .macOS("13.0"),
-    .iOS("16.0"),
-    .watchOS("9.0"),
-    .tvOS("16.0"),
-    .custom("visionos", versionString: "1.0"),
+    .macOS(.v13),
+    .iOS(.v16),
+    .watchOS(.v9),
+    .tvOS(.v16),
+    .visionOS(.v1),
```

Comment:
Thank you for contributing!

Note that this last one, for visionOS, required using `.custom` originally because the version of Xcode that our macOS CI system was using at that time did not yet have that API. It appears it is new enough now, but I recall there was a second usage of `.custom` later in this same file. Are you interested in updating that one from `.custom("visionos")` to `.visionOS` as well?

Reply:
I had missed it. Corrected. https://github.com/apple/swift-testing/pull/40/commits/ab019df73cd6199b18ecca16e7c8ddedaa37fc72